src/iexamine.cpp

```diff
@@ -3535,45 +3596,57 @@ void iexamine::autodoc( player &p, const tripoint &examp )
             const itype *itemtype = it->type;
             const bionic_id &bid = itemtype->bionic->id;
 
-            if( p.has_bionic( bid ) ) {
-                popup( _( "You have already installed this bionic."  ) );
+            if( patient.is_npc() && !bid->npc_usable ) {
+                //~ %2$s is the bionic CBM display name, %2$s is the patient name
+                popup( _( "%1$s cannot be installed on npc %2$s"), it->display_name().c_str(), patient.name );
```

Comment:
Suggest either:
removing `npc`
or
changing to: `cannot be installed on follower %2$s` as currently non-followers (hostile, neutral, or followers "on guard duty"/assigned to base) can't be operated on.

First option may be more prudent for future development (operations may take place on captured NPCs or whatever else)

Reply:
Fixed.